modules/git/repo_language_stats_gogit.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
```

Comment:
Seems like the touched gogit code is mostly duplicate. Move to a shared file?

Reply:
The differences are `continue / return nil`, `.Name() / .Name`, `.Size() / .Size` and content reading which makes it not as easy as it looks like. I would prefer to leave it as it is for this PR.

Reply:
Ok, should be refactored later.